README.md

```diff
@@ -174,15 +173,35 @@ When this VM is first created Chrome and Firefox will be upgraded to the latest
 
 All of these libraries are already bundled within the Infusion image.
 
+## How Do I Run Tests? ##
+
+There are two options available for running tests. The first option involves using browsers installed on your computer and the second uses browsers available in a VM.
+
+### Run Tests Using Browsers Installed On Your Computer ###
+
+Using this option requires the installation of [Testem](https://github.com/testem/testem/#installation) and then running ``testem ci --file tests/testem.json`` in this directory. Any browsers that Testem finds on your platform will be launched sequentially with each browser running the full Infusion test suite. The results will be returned in your terminal in the [TAP](https://testanything.org/) format. You can use the ``testem launchers`` command to get a list of available browsers.
+
+Please note that any browsers launched will need to remain in the foreground because some of the tests require window focus.
```

Comment:
This statement could use clarification. If understand correctly, the user needs to actively monitor the tests as they run and focus each browser as it is opened. Is that correct?